cluster-autoscaler/core/utils/utils.go

```diff
@@ -28,9 +28,17 @@ import (
 	"k8s.io/autoscaler/cluster-autoscaler/utils/gpu"
 )
 
+const (
+	// VirtualNodeLabelValue is the value of the label that is set on virtual kubelet
+	VirtualNodeLabelValue = "virtual-kubelet"
+)
+
 // isVirtualNode determines if the node is created by virtual kubelet
 func isVirtualNode(node *apiv1.Node) bool {
-	return node.ObjectMeta.Labels["type"] == "virtual-kubelet"
+	if node == nil {
```

Comment:
`return node != nil && node.ObjectMeta.Labels["type"] == VirtualKubeletNodeLabelValue` is more concise